Orange/widgets/data/tests/test_owoutliers.py

```diff
@@ -130,13 +131,17 @@ def test_nans(self):
 
     def test_in_out_summary(self):
         info = self.widget.info
+        data = self.iris
+        input_sum = self.widget.info.set_input_summary = Mock()
+        output_sum = self.widget.info.set_output_summary = Mock()
```

Comment:
There is not much point in `self.assertEqual(info._StateInfo__input_summary.brief, "")` lines, since `self.widget.info.set_input_summary` was mocked.

Reply:
The same goes for the code some lines below.
`info._StateInfo__input_summary.brief` could never have been changed, since mocked, so there is no point of testing it.

You could leave the test as it was, just add the lines to test the detail:
`self.assertEqual(info._StateInfo__input_summary.detail, detail)`